src/libsync/owncloudpropagator.h

```diff
@@ -188,18 +188,18 @@ class OWNCLOUDSYNC_EXPORT PropagateDirectory : public PropagatorJob {
     QScopedPointer<PropagateItemJob>_firstJob;
 
     // all the sub files or sub directories.
-    QVector<PropagatorJob *> _subJobs;
+    QList<PropagatorJob *> _subJobs;
```

Comment:
Not sure if this is good, https://marcmutz.wordpress.com/effective-qt/containers/
@ogoffart might have to say more